awswrangler/neptune/neptune.py

```diff
@@ -0,0 +1,134 @@
+from awswrangler.neptune import client
+from typing import Any
+import pandas as pd
+
+def read_gremlin(
+    client: client,
+    traversal: str
+) -> pd.DataFrame:
+    """Return results of a Gremlin traversal as pandas dataframe.
+
+    Parameters
+    ----------
+    client : neptune.Client
+        instance of the neptune client to use
+    traversal : str
+        The gremlin traversal to execute
+
+    Returns
+    -------
+    Union[pandas.DataFrame, Iterator[pandas.DataFrame]]
+        Results as Pandas DataFrame
+
+    Examples
+    --------
+    Run a Gremlin Query
+
+    >>> import awswrangler as wr
+    >>> client = wr.neptune.Client(host='NEPTUNE-ENDPOINT')
+    >>> df = wr.neptune.gremlin.read(client, "g.V().limit(5).valueMap()")
```

Comment:
I assume this should be `wr.neptune.read_gremlin` instead?

Reply:
I would vote for wr.neptune.gremlin.read may be, because each graph might have their own functions and it creates a good isolation between property graph and RDF.

Reply:
I would vote for wr.neptune.gremlin.read may be, because each graph might have their own functions and it creates a good isolation between property graph and RDF.

Reply:
I went back and forth on this a bit and I ended up with `read_gremlin` and `read_opencypher` instead of creating subclasses for each language, as that seemed to better fit the overall patterns in the library.  I must have missed that in the docs, but they have been updated now.

Reply:
Agreed would prefer dedicated methods like above